capstone/src/main/java/com/google/sps/servlets/EventListing.java

```diff
@@ -34,7 +34,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/eventlisting.html");
   }
 
-  public Event createEvent(VotingEvent votingEvent) throws IOException {
+  public static Event createEvent(VotingEvent votingEvent) throws IOException {
```

Comment:
Optional: instead of making this static, you could add a package private constructor to this class and create an instance of it for each test.

Reply:
Hmm, since EventListingTest is outside of the EventListing package I wouldn't be able to access the constructor right?

Reply:
You should probably move the test into a servlets/ directory so that it mirrors this path exactly. Then you'll just update the package and you will be able to access EventListing.

If you want to get this in, feel free to submit as is, but it shouldn't take long to complete the fix.